lib/web/apps.go

```diff
@@ -362,3 +356,20 @@ func (h *Handler) proxyDNSNames() (dnsNames []string) {
 	}
 	return dnsNames
 }
+
+// extractAppsWithoutTCPEndpoint iterates over a list of app servers and extracts
+// its application without its URI prefixed with "tcp". It also keeps a
+// counter of how many TCP applications were excluded from the list (e.g. used to
+// subtract it from the TotalCount received from ListResources api).
+func extractAppsWithoutTCPEndpoint(appServers []types.AppServer) (apps types.Apps, numExcluded int) {
+	for _, server := range appServers {
+		// Skip over TCP apps since they cannot be accessed through web UI.
+		if !server.GetApp().IsTCP() {
+			apps = append(apps, server.GetApp())
+		} else {
+			numExcluded++
+		}
+	}
+
+	return apps, numExcluded
```

Comment:
```suggestion
	for _, server := range appServers {
		// Skip over TCP apps since they cannot be accessed through web UI.
		if !server.GetApp().IsTCP() {
			apps = append(apps, server.GetApp())
		} 
	}

	return apps, len(appServers) - len(apps)
```